src/oop/programs_with_multiple_classes/hard/q5/Bookcase.java

```diff
@@ -0,0 +1,19 @@
+package oop.programs_with_multiple_classes.hard.q5;
+
+public class Bookcase {
+    private int sizeCapacity;
+    private Book b1;
```

Comment:
You can probably even make this an array of book objects.

Reply:
Okay, but how should the determineTime method work then?